Refine typography and font styles in Page and Skeleton layouts

maribelferreira opened the pull request:
- Adjust prose text sizes for headings (h1, h2, h3)
- Increase Inter font weight to 700
- Enhance text color for strong elements and list markers
- Fine-tune heading margins and spacing

maribelferreira committed:
Refine typography and font styles in Page and Skeleton layouts

- Adjust prose text sizes for headings (h1, h2, h3)
- Increase Inter font weight to 700
- Enhance text color for strong elements and list markers
- Fine-tune heading margins and spacing

gnarea committed:
Merge branch 'main' into markdown-layout-polish

gnarea commented on src/layouts/Page.astro:
```diff
@@ -30,17 +30,19 @@ const articleClasses = [
   "prose-h2:mb-4",
-  "prose-h3:text-2xl",
-  "prose-h3:mt-10",
+  "prose-h3:text-[1.25rem]",
```
I'd suggest not to use arbitrary values

gnarea reviewed (changes requested):
Much better! However, `code` is now bigger than the rest of the text:

![Screenshot_20250309_191815](https://github.com/user-attachments/assets/cc52b59f-f64f-42b0-81d7-90c33a32c541)

gnarea commented:
> code is now bigger than the rest of the text

never mind, I think it's just the padding that makes it look bigger

gnarea commented:
Another issue I noticed is that links inside `<code>` aren't being highlighted. See: https://veraid.net/overview/#core-libraries

maribelferreira committed:
Refine inline code and image styles in markdown layout

- Reduce inline code padding and text size
- Center and constrain image width with max-width
- Maintain consistent typography approach

maribelferreira committed:
Adjust heading styles and colors in markdown layouts

- Set global h1-h6 text color to white in Skeleton layout
- Remove redundant white color classes from Page layout

maribelferreira committed:
Enhance code link styles in markdown layout

- Replace arbitrary text sizes with Tailwind's predefined sizes for h1 and h3
- Add custom styles for links within code blocks and code within links
- Use green color variants for improved readability and interaction

maribelferreira committed:
Remove hardcoded white color from h6 headings

maribelferreira committed:
Add white text color to markdown headings in Page layout

gnarea committed:
Merge branch 'main' into markdown-layout-polish
